### PR TITLE
open-cluster-management/backlog#7709 clusterroles

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: manage_cluster/define_clusterrole.adoc [hunk 1]
```diff
@@ -0,0 +1,81 @@
+[#configuring-a-specific-cluster-management-role]
+= Configuring a specific cluster management role
+
+When you install {product-title}, the default configuration provides the `cluster-admin` role on the {product-title-short} hub cluster. This permission enables you to create, manage, and import managed clusters on the hub cluster. In some situations, you might want to limit the access to certain `ManagedClusterSets` that are managed by the hub cluster, rather than access to all of the managed clusters on the hub cluster.
```
Comment (skeeey) [inline manage_cluster/define_clusterrole.adoc]: This should be `ManagedCluster` not `ManagedClusterSets`

File: manage_cluster/define_clusterrole.adoc [hunk 2]
```diff
@@ -0,0 +1,81 @@
+[#configuring-a-specific-cluster-management-role]
+= Configuring a specific cluster management role
+
+When you install {product-title}, the default configuration provides the `cluster-admin` role on the {product-title-short} hub cluster. This permission enables you to create, manage, and import managed clusters on the hub cluster. In some situations, you might want to limit the access to certain `ManagedClusterSets` that are managed by the hub cluster, rather than access to all of the managed clusters on the hub cluster. 
+
+You can limit access to certain `ManagedClusterSets` by defining a `clusterrole` and applying it to a user or group. Complete the following steps to configure and apply a role:
```
Comment (skeeey) [inline manage_cluster/define_clusterrole.adoc]: This should be `ManagedCluster` not `ManagedClusterSets`

File: manage_cluster/define_clusterrole.adoc [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  - managedclusters/accept
+  resourceNames:
+  - <managed-cluster-name>
+  verbs:
+  - update
+  ----
+  +
+  Replace `_clusterrole-name_` with the name of the cluster role that you are creating.
+  +
+  Replace `_managed-cluster-name_` with the name of the managed cluster or `ManagedClusterSet` that you want the user to have access to.
```
Comment (skeeey) [inline manage_cluster/define_clusterrole.adoc]: no `ManagedClusterSet `

File: manage_cluster/define_clusterrole.adoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  - <managed-cluster-name>
+  verbs:
+  - update
+  ----
+  +
+  Replace `_clusterrole-name_` with the name of the cluster role that you are creating.
+  +
+  Replace `_managed-cluster-name_` with the name of the managed cluster or `ManagedClusterSet` that you want the user to have access to. 
+  
+  . Apply the clusterrole definition by entering the following command:
```
Comment (dockerymick) [inline manage_cluster/define_clusterrole.adoc]: Can you add code back-ticks around clusterrole

**Comment by cadawson** [comment]
Hello @skeeey . I addressed your comments in the document. I think we are ready to go. Would you like to take a final look at it and approve, please? Thank you!

**Review by skeeey — APPROVED** [review]

**Comment by skeeey** [comment]
done

**Pull request merged**